src/interactor/MemberInteractor.java

```diff
@@ -0,0 +1,24 @@
+package interactor;
+
+import boundary.IMember;
+import entity.MemberTask;
+import entity.Profile;
+import gateway.IGateway;
+
+public class MemberInteractor implements IMember {
+    public IGateway gatewayInteractor;
```

Comment:
```suggestion
    public IGateway gateway;
```

Reply:
A gateway is not an interactor. A gateway is a boundary to an external data source. That external data source is the ProjectStateManager. An interactor is a collection of use cases for a specific actor. The member interactor is a collection of member use cases such as "View Member Task".

Reply:
I see. This explanation is really helpful - clarifies a lot of things with respect to the architecture. Thanks.